editor/project_settings_editor.cpp

```diff
@@ -298,6 +304,10 @@ void ProjectSettingsEditor::_add_feature_overrides() {
 		ee->get_export_platform(i)->get_platform_features(&p);
 		for (const String &E : p) {
 			presets.insert(E);
+			if (E == "linuxbsd") {
+				presets.insert("linux");
+				presets.insert("bsd");
+			}
```

Comment:
Perhaps we should keep only `linuxbsd` in the list (without `linux`, `bsd` and other "sub-platforms")? Project Settings UI now allows feature tags that are not in the list.